doc/deploy/android.md

```diff
@@ -1,33 +1,129 @@
-## Deploying to the Google Play Store
-##### Prerequirements
-- You **must** read the [general deploy info](README.md).
-- You **must** have the official PVTrAck signing key. See [the keys doc](android-keys.md) for more info.
-- You **must** have the Android SDK and Android build tools installed (both bundled with Android Studio). We target API level 25 (Nougat, 7.1).
-- You **must** have access to the [Google Play Developer Console](play.google.com/apps/publish) (see @sherson).
-- You *should* have [Android Studio](https://developer.android.com/studio/index.html) installed.
+# Building an Android App to Deploy in Google Play
 
-##### Steps
+## Requirements
+_These instructions assume you are running a Mac. If you're not, the installation
+instructions will be vastly different. You can ask for help or, since you're probably
+running Linux, just translate them like you're probably used to doing._
 
-1. `npm install`
+### General
+- Did you read the [general deploy info](README.md)?
+- Do you have access to the [Google Play Developer Console](play.google.com/apps/publish)?
+ If not, see Adam.
 
-2. Open `config.xml`. On line 2, increment `android-versionCode` with your appropriately chosen new version number.
+### Android Studio
+We use Android Studio to easily manage the Android SDK tools. If you wish, you can attempt
+to download the tools separately, but these instructions will not help you because I found the
+attempt maddening. Plus, Android Studio will keep track of updates for you, which is nice.
+1. Get [Android Studio](https://developer.android.com/studio/)
+1. Use it to download the SDK platforms and tools: 
+  1. Click menu items Android Studio -> Appearance and Behavior -> System Settings -> Android SDK
```

Comment:
Instead of menu items, should we say Android Studio > Preferences > Appearance and Behavior ...? That's a bit clearer to me.

Reply:
Ah yeah I was totally unclear. I meant "click the menu items "This" -> "that" -> "this"", not "click a button called 'menu items'. Will change so that's clearer.